snap_plugin/v1/plugin.py

```diff
@@ -284,9 +302,15 @@ def __init__(self):
 
         flags = [
             ("config", FlagType.value, "JSON Snap global config"),
+            ("port", FlagType.value, "GRPC server port"),
             ("stand-alone", FlagType.toggle, "enable stand alone mode"),
             ("stand-alone-port", FlagType.value, "http port for stand alone mode", 8181),
-            ("log-level", FlagType.value, "logging level 0:panic - 5:debug", 3)
+            ("log-level", FlagType.value, "logging level 0:panic - 5:debug", 3),
+            ("tls-enabled", FlagType.toggle, "enable tls"),
```

Comment:
Please be compatible with snap documentation:
https://github.com/intelsdi-x/snap/blob/master/docs/SECURE_PLUGIN_COMMUNICATION.md#enabling-secure-communication
I think those flags should be named same as in documentation